agent/resources/cgroup/cgroup_controller_linux.go

```diff
@@ -50,32 +50,44 @@ func (c *control) Create(cgroupSpec *Spec) (cgroups.Cgroup, error) {
 
 	// Create cgroup
 	seelog.Infof("Creating cgroup %s", cgroupSpec.Root)
-	control, err := c.New(cgroups.V1, cgroups.StaticPath(cgroupSpec.Root), cgroupSpec.Specs)
+	controller, err := c.New(cgroups.V1, cgroups.StaticPath(cgroupSpec.Root), cgroupSpec.Specs)
 
 	if err != nil {
 		return nil, errors.Wrapf(err, "cgroup create: unable to create controller")
 	}
 
-	return control, nil
+	return controller, nil
 }
 
 // Remove is used to delete the cgroup
 func (c *control) Remove(cgroupPath string) error {
 	seelog.Debugf("Removing cgroup %s", cgroupPath)
 
-	control, err := c.Load(cgroups.V1, cgroups.StaticPath(cgroupPath))
```

Comment:
I wanted to avoid any naming `confusion|conflicts`.
`cgroup` is also the name of the package and we also have a `containerd/cgroups` package. 

Should we shrink it down to something like `ctl` or `cgrp` or `cg` ?